index.js

```diff
@@ -0,0 +1,207 @@
+const select = document.querySelector("select");
+const section = document.querySelector("#display-info");
+const form = document.querySelector("form");
+
+const getFilms = async () => {
+  try {
+    const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+    debugger;
+    res.data.forEach((film) => {
+      const option = document.createElement("option");
+      option.innerText = film.title;
+      select.appendChild(option);
```

Comment:
we can use the value property for options and assign it the id of the movie, so that we can make a request to a single film later.
`option.value = film.id`